docs/product/performance/filters-display/widgets.mdx

```diff
@@ -4,67 +4,99 @@ sidebar_order: 10
 description: "Learn more about Sentry's Performance widgets such as Most Related Issues, Most Regressed Transactions, and Slow HTTP Ops, and how they can be used to surface performance insights in your applications."
 ---
 
-Performance widgets offer visualizations that you can change to best match your workflow. They provide performance-specific views that allow you see and act on transaction data across your organization.
-Depending on which **Performance** tab you've selected, different curated performance widgets will be available.
+Performance widgets offer visualizations that you can change to best match your workflow. They provide performance-specific views that allow you to see and act on transaction data across your organization.
+Depending on which [Performance Tab](/product/performance/filters-display/#tabs) you've selected, different curated performance widgets will be available.
 
 <Note>
 
 The value displayed in the top right of some widgets is the overall value for the selected time period, rather than the average of the data displayed in the chart. For datasets with low event volume, the overall value may be higher or lower than expected because metrics like [Apdex](/product/performance/metrics/#apdex) and [User Misery](/product/performance/metrics/#user-misery) are non-linear.
```

Comment:
```suggestion

The value displayed in the top right of some widgets is the **overall value for the selected time period**, not the average of the data displayed in the chart. For datasets with low event volume, the overall value may be higher or lower than expected because metrics like [Apdex](/product/performance/metrics/#apdex) and [User Misery](/product/performance/metrics/#user-misery) are non-linear.
```